go/v1beta1/storage/types.go

```diff
@@ -74,3 +74,46 @@ type esBulkResponse struct {
 type esBulkResponseItem struct {
 	Index *esIndexDocResponse `json:"index,omitempty"`
 }
+
+// Elasticsearch /_msearch query fragments
+
+type esMsearchQueryFragment struct {
+	Index string `json:"index"`
+}
+
+// Elasticsearch /_msearch response
+
+type esMsearch struct {
+	Responses []esMsearchResponse `json:"responses"`
+}
+
+type esMsearchResponse struct {
+	Hits esMsearchResponseHits `json:"hits"`
+}
+
+type esMsearchResponseHits struct {
+	Total esSearchResponseTotal         `json:"total"`
+	Hits  []esMsearchResponseNestedHits `json:"hits"`
+}
+
+type esMsearchResponseNestedHits struct {
+	Source esMsearchResponseSource `json:"_source"`
+}
+
+type esMsearchResponseSource struct {
```

Comment:
what's the idea behind this struct?  we can't expect every response to always look like this.  this will only specifically work for vulnerability occurrences.